receiptui/src/main/java/com/hyperwallet/android/ui/receipt/viewmodel/ProgramModel.java

```diff
@@ -0,0 +1,7 @@
+package com.hyperwallet.android.ui.receipt.viewmodel;
+
+public enum ProgramModel {
```

Comment:
add this in `common` module

Reply:
@peter-joseph : Moved ProgramModel to the 'view' package in common-ui.